src/Node/Stream/StdIO.purs

```diff
@@ -0,0 +1,11 @@
+-- | The standard IO streams for the current process.
+module Node.Stream.StdIO where
```

Comment:
Actually, on second thoughts, yes, I think a `node-process` library is a better fit for these. Shall I remove these, and then I can create one once this is merged and released?